interface/main/holidays/import_holidays.php

```diff
@@ -0,0 +1,197 @@
+                </td>
+                <td class='detail' nowrap>
+                    <input type="file" name="form_file" size="40" />
+                </td>
+            </tr>
+            <tr>
+                <td class='detail' nowrap>
+                    <?php echo xlt('File on server (modification date)'); ?>
+                </td>
+                <td class='detail' nowrap>
+                    <?php
+                    if(!empty($csv_file_data)){?>
+
+                        <?php $path=explode("/",$holidays_controller->get_target_file());?>
+                        <?php $filename=$path[count($path)-1];?>
+                        <?php unset($path[count($path)-1]);?>
+
+                        <?php
+                        // build the path to the for the download request
+                        $phppagepath = "http://$_SERVER[HTTP_HOST]$_SERVER[REQUEST_URI]?filename=";?>
+                        <a href="#" onclick='window.open("import_holidays.php?download_file=1")'><?php echo text($csv_file_data['date']);?></a>
+                    <?php }else{
+                        echo htmlspecialchars(xl('File not found'));
+                    }?>
+                </td>
+            </tr>
+
+        <tr bgcolor='#dddddd'>
+                <td align='center' class='detail' colspan='2'>
+                    <input type='submit' name='bn_upload' value='<?php echo xla('Upload / Save') ?>' />
+                </td>
+            </tr>
+        </table>
+        </p>
+</form>
+        <table border='1'>
+
+        <tr >
+            <td >
+                <form method='post' action='import_holidays.php' onsubmit='return top.restoreSession()'>
+                    <input type='submit' name='import_holidays' value='<?php echo xla('Import holiday events') ?>'></br>
+
+                </form>
+            </td>
+
+            <td>
+                    <?php echo xla('CSV to calendar_external table'); ?></br>
+                <?php echo xla('If  the csv has been uploaded please click on "Import holiday events" button
+                    **NOTE: clicking on the button will remove all the existing rows in the calendar external table')?>
+                </td>
+        </tr>
+            <tr >
+                <td >
+                    <form method='post' action='import_holidays.php' onsubmit='return top.restoreSession()'>
+                        <input type='submit' name='sync' value='<?php echo xla('Syncronize') ?>' /></br>
+                    </form>
+                </td>
+                <td >
+                    <?php echo xla('calendar_External to events'); ?></br>
+                    <?php echo xla('If you have already filled the calendar external table please click on "Syncronize" to have the holidays n the calendar view 
+                    **NOTE: clicking on the button will remove all the existing rows in the events table related to holidays')?>
+                </td>
```

Comment:
Note that the 4 above xla() calls should be xlt() since they are not within html attributes.